transit_odp/organisation/csv/overall.py

```diff
@@ -89,6 +95,54 @@
     "string_lines",
 )
 
+DATACATALOGUE_ATTRIBUTE_FIELDS = (
+    "fares_metadata_id",
+    "xml_file_name",
+    "national_operator_code",
+    "line_name",
+)
+
+
+def get_fares_data():
+    """
+    Returns list of fares information (xml file name, NOCs and line name)
+    when dataset_id in DatasetRevision table match
+    fares_metadata_id in DataCatalogueMetaData table.
+    """
+    dataset_ids_dict = (
+        DataCatalogueMetaData.objects.filter(
+            fares_metadata_id__in=FaresMetadata.objects.all().values_list(
+                "datasetmetadata_ptr"
+            )
+        )
+        .order_by("fares_metadata_id")
+        .values(*DATACATALOGUE_ATTRIBUTE_FIELDS)
+    )
+
+    for dataset_id in dataset_ids_dict:
```

Comment:
dataset_id is a horrible name, what you have is a QuerySet that returns a dict not an integer:
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#django.db.models.query.QuerySet.values